plant/plants/requests.py

```diff
@@ -0,0 +1,19 @@
+import environ
+import requests
+
+
+def fetch_plant_families():
```

Comment:
Type hint for return value would be helpful as name does not say if we return list of Plant objects, Plant queryset or strings